react/features/toolbox/components/web/Toolbox.js

```diff
@@ -976,8 +988,20 @@ class Toolbox extends Component<Props, State> {
                     onClick = { this._onToolbarOpenKeyboardShortcuts }
                     text = { t('toolbar.shortcuts') } />,
             this._shouldShowButton('help')
-                && <HelpButton
+                && <OpenUrlButton
```

Comment:
I really don't like how vebose this becomes everywhere we need to use this. Please create a DownloadApps button instead.

Reply:
Done, please revisit.